Aptoide/appcoins-wallet-android
# [APPC-1416] Balance on checkout screen implementation

**What does this PR do?**

   Adds balance to the merged appcoins view 

**Database changed?**

   No

**Where should the reviewer start?**

MergedAppcoinsFragment.kt

**How should this be manually tested?**

Check different use cases: No balance, no balance with one and balance with other, both with balance, high balance

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1416

**Code Review Checklist**
